lib/LaTeXML/resources/XSLT/LaTeXML-webpage-xhtml.xsl

```diff
@@ -134,294 +134,26 @@
-
-  <xsl:template name="multiscripts">
-    <xsl:param name="i"/>
-    <xsl:param name="n"/>
-    <xsl:param name="op"/>
-    <xsl:param name="other"/>
-    <xsl:if test="$i &lt;= $n">
-      <xsl:apply-templates select="*[$i]" mode="visible-script">
-        <xsl:with-param name="op" select="$op"/>
-      </xsl:apply-templates>
-      <xsl:call-template name="multiscripts">
-        <xsl:with-param name="i" select="$i+1"/>
-        <xsl:with-param name="n" select="$n"/>
-        <xsl:with-param name="op" select="$other"/>
-        <xsl:with-param name="other" select="$op"/>
-      </xsl:call-template>
-    </xsl:if>
-  </xsl:template>
-
-  <xsl:template match="m:mfrac" mode="visible-text">
-    <xsl:apply-templates select="*[1]" mode="visible-nested"/>
-    <xsl:text>/</xsl:text>
-    <xsl:apply-templates select="*[2]" mode="visible-nested"/>
-  </xsl:template>
-
-  <xsl:template match="m:mphantom" mode="visible-text"/>
-  <xsl:template match="m:mspace" mode="visible-text">
-    <xsl:text> </xsl:text>
-  </xsl:template>
-
-  <xsl:template match="m:mfenced"  mode="visible-text">
-    <xsl:value-of select="@open | '('"/>
-    <xsl:apply-templates mode="visible-text-punctuated"/>
-    <xsl:value-of select="@close | ')'"/>
-  </xsl:template>
-
-  <!-- NO, I'm not going to try to decipher @separators -->
-  <xsl:template match="*"  mode="visible-text-punctuated">
-    <xsl:apply-templates mode="visible-text"/>
-    <xsl:if test="./following-sibling::*">
-      <xsl:text>,</xsl:text>
-    </xsl:if>
-  </xsl:template>
-
-  <xsl:template match="m:mtable"  mode="visible-text">
-    <xsl:text>[</xsl:text>
-    <xsl:apply-templates mode="visible-text"/>
-    <xsl:text>]</xsl:text>
-  </xsl:template>
-
-  <xsl:template match="m:mtr"  mode="visible-text">
-    <xsl:text>[</xsl:text>
-    <xsl:apply-templates mode="visible-text"/>
-    <xsl:text>]</xsl:text>
-  </xsl:template>
-
-  <xsl:template match="m:mtd"  mode="visible-text">
-    <xsl:apply-templates mode="visible-text"/>
-    <xsl:if test="./following-sibling::m:mtd">
-      <xsl:text>,</xsl:text>
-    </xsl:if>
-  </xsl:template>
-
-  <xsl:template match="m:menclose"  mode="visible-text">
-    <xsl:value-of select="@notation"/>
-    <xsl:text>(</xsl:text>
-    <xsl:apply-templates mode="visible-text"/>
-    <xsl:text>)</xsl:text>
-  </xsl:template>
-
-  <xsl:template match="m:msqrt" mode="visible-text">
-    <xsl:text>sqrt(</xsl:text>
-    <xsl:apply-templates mode="visible-text"/>
-    <xsl:text>)</xsl:text>
-  </xsl:template>
-
-  <xsl:template match="m:mroot" mode="visible-text">
-    <xsl:text>root(</xsl:text>
-    <xsl:apply-templates select="*[1]" mode="visible-text"/>
-    <xsl:text>,</xsl:text>
-    <xsl:apply-templates select="*[2]" mode="visible-text"/>
-    <xsl:text>)</xsl:text>
-  </xsl:template>
-
-  <xsl:template match="m:merror" mode="visible-text">
-    <xsl:text>ERROR(</xsl:text>
-    <xsl:apply-templates mode="visible-text"/>
-    <xsl:text>)</xsl:text>
-  </xsl:template>
-
```

Comment:
I take it this big red deletion from the XSL was redundant to the new UnicodeMath post-processor? Great if so, good to see the XSLT "feature programming" shrinking.

Reply:
Yeah, isn't it lovely :>